scripts/bootstrap-ansible.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
-
-# minimal bootstrapping before kicking off ansible:
-#  install only what ansible needs to survive, or doesn't know how to do
-sudo dnf -y install python2 python2-dnf libselinux-python
```

Comment:
Whether or not this is still necessary, it isn't being called run by anything on this branch and hasn't been since November 2016.